### DISCUSSION TITLE
Is the 1.19.2 fabric version compatible with 1.19.4 also?

### DISCUSSION THREAD

**Discussion by ButterKing-28** [mortuusars/Exposure#36] [Q&A] [answered]
I am a huge photography nerd (and I'm going for film production in college currently) so I was AMAZED when I found this mod! Is the 1.19.2 fabric version compatible with 1.19.4 also?

**Comment by mortuusars** [comment, marked as answer]
I have not tested it, but 99.9% that it is not compatible.
I have to develop features on, and maintain 4 versions of the mod. With 1.19.4 support it would be 6 (forge and fabric).
It's a pain to have multiple versions to develop on, and maintain. I don't have exact numbers, but, from my experience, side versions (like 1.19.4, 1.20.2) have at most 1/10 of the players. So it's not worth the hassle to support them.
